packages/admin/admin/src/mui/menu/Menu.tsx

```diff
@@ -17,21 +80,28 @@ export interface MenuProps {
     permanentDrawerPaperProps?: PaperProps;
 }
 
-const MenuDrawer: React.FC<WithStyles<typeof styles> & MenuProps> = ({
-    classes,
-    children,
-    drawerWidth = 300,
-    variant = "permanent",
-    temporaryDrawerProps = {},
-    permanentDrawerProps = {},
-    temporaryDrawerPaperProps = {},
-    permanentDrawerPaperProps = {},
-}) => {
+export function Menu(inProps: MenuProps) {
+    const {
+        children,
+        drawerWidth = 300,
+        variant = "permanent",
+        temporaryDrawerProps = {},
+        permanentDrawerProps = {},
+        temporaryDrawerPaperProps = {},
+        permanentDrawerPaperProps = {},
+        slotProps,
```

Comment:
`temporaryDrawerPaperProps` and `permanentDrawerPaperProps` can probably be deprecated or removed in favor of `slotProps`, but there should be two separate slots then, one for each `Drawer`.

Reply:
@jamesricky as discussed before, I didn't change the usage of `temporaryDrawerPaperProps ` and `permanentDrawerPaperProps`. Instead I separated the `Drawer`-slot to two different ones: `temporaryDrawer`& `permanentDrawer`